open_academy/security/ir.model.access.csv

```diff
@@ -1,3 +1,4 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
 access_course,course,model_course,base.group_user,1,1,1,1
 access_session,session,model_session,base.group_user,1,1,1,1
+open_academy.access_wizard_session,access_wizard_session,open_academy.model_wizard_session,base.group_user,1,1,1,1
```

Comment:
why open_academy.access_wizard_session, better access_wizard_session

Reply:
Also could be better to add a more descriptive name as access_open_academy_add_attendee_sessions